src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -36,3 +39,17 @@ export const combinedDerivedSortedNetworkTokensUi: Readable<TokenUi[]> = derived
 			$exchanges
 		})
 );
+
+/**
+ * All user-enabled tokens matching the selected network or chain fusion that do not have an index canister.
+ */
+export const enabledNetworkTokensWithoutIndexCanister: Readable<Token[]> = derived(
```

Comment:
- Is there plan to reuse this information anywhere else? If not, I would scope the information to the component. I think a lean approach about adding new stores is for the best given our performance issues.

- Why considering ETH and BTC? Should you just iterate on `enabledIcrcTokens` or `enabledIcrcTokensNoCk`?

Reply:
@peterpeterparker 
No, at the moment it is not planed to reuse it anywhere else. I guess you are right, i should move the implementation into the component. 

Why not? Isn't it possible that an ETH or BTC does not have an Index canister?

Reply:
> No, at the moment it is not planed to reuse it anywhere else. I guess you are right, i should move the implementation into the component.

I would say let's do that if that works for you?

> Isn't it possible that an ETH or BTC

Canisters only existing on the Internet Computer that's why I'm wondering why we are considering Ethereum and Bitcoin, unless e.g. ckBTC or ckETH are listed explicitely within this derived stores but, I don't think it's the case?

Reply:
@peterpeterparker 
That's ok for me, i just moved the implementation into the component.

Hmm, i guess you are right.